src/components/user-card/UserCard.js

```diff
@@ -18,6 +18,7 @@ export const UserCard = (props) => {
     if (!user) return null;
     const { first_name, last_name, email, address, _links } = user;
     return (
+
         <div className="may-user-card card">
             <img src={_links.avatar.href} className="may-user-card-avatar rounded-circle" alt="image"/>
```

Comment:
you can go deeper with destructuring of your href prop
    const { first_name, last_name, email, address, _links: {avatar: {href}} } = user;
```            <img src={href} className="may-user-card-avatar rounded-circle" alt="image"/>
```

but that's not required, just a thing, that many developers consider as a good practice